core/src/main/java/org/apache/calcite/tools/RelBuilder.java

```diff
@@ -487,8 +487,8 @@ public RexNode field(int inputCount, String alias, String fieldName) {
                 p.e.right.getName()));
       }
     }
-    throw new IllegalArgumentException("no aliased field found; fields are: "
-        + fields);
+    throw new IllegalArgumentException("{alias=" + alias + ",fieldName=" + fieldName + "} "
```

Comment:
nit: space after `,` in `,fieldName=`.

Reply:
Thanks for the feedback @hsyuan. I prefer to leave it as it is though, I am just using the same format that is being used a few lines before to store `fields` information.

Reply:
Ah, I found it. then it is ok.